deny.toml

```diff
@@ -39,8 +39,8 @@ skip = [
     { name = "digest", version = "=0.9.0" },
     { name = "sha2", version = "=0.9.9" },
 
-    # near-vm-runner and wasmer-compiler-near use 0.78.2
-    { name = "wasmparser", version = "=0.78.2" },
+    # wasmtime uses 0.84.0
```

Comment:
Is there a reason for us to not update wasmtime? The most recent version (3.0) depends on `0.93`.

Reply:
Not really; it’s just that it felt like something unrelated to the topic at-hand